src/cpp/redis.cpp

```diff
@@ -144,7 +117,7 @@ CommandReply Redis::put_tensor(TensorBase& tensor)
 
 CommandReply Redis::get_tensor(const std::string& key)
 {
-    Command cmd;
+    GetTensorCommand cmd;
     cmd.add_field("AI.TENSORGET");
```

Comment:
These could be moved to the GetTensorCommand constructor. Similar with other specialized commands